docs/UserGuide.md

```diff
@@ -803,8 +835,9 @@ Example with the following assignment.json file:
   ]
 }
 ```
-`addGrade n/JohnDoe a/Ex01 s/5` will add an assignment name
+`addGrade name/JohnDoe assignment/Ex01 score/5` will add an assignment name
 Assignment01 with score 5 to contact JohnDoe.
```

Comment:
This line seems to have an error too. It is supposed to be "will add an assignment name Ex01.." instead of "will add an assignment name Assignment01.."